Class/Tools.php

```diff
@@ -391,7 +512,15 @@ public static function htmlRadio($name, $input, $value = null, $options = array(
         return mb_substr($result, mb_strlen(@$options['separator']));
     }
 
-    // HTML notation for the <textarea> tag. See html_textinput() for more info.
+    /** 
+     * HTML notation for the <textarea> tag. See html_textinput() for more info.
+     *
+     * @param string $name
+     * @param string $content
+     * @param int $cols (optional)
+     * @param int $rows (optional)
+     * @param mixed[] $options (optional) See self::htmlTextInput() for more info
+     */
     public static function htmlTextarea($name, $content, $cols = 60, $rows = 5, $options = array())
     {
         $label = @$options['label'];
```

Comment:
A když $options['label'] nebude definovaný a chybu to nevyhodí, tak $label bude obsahovat co?

Reply:
Díky. Opět opravuju. (btw, v tomhle případě by obsahovala NULL)